snuba/datasets/events_processor_base.py

```diff
@@ -30,7 +30,7 @@
 logger = logging.getLogger(__name__)
 
 
-class ReplacementTypes(str, Enum):
```

Comment:
Why rename this? It really clutters up your changes.

Reply:
Enum generally is singular, it seemed weird to have `replacement_type: ReplacementTypes` so just updated to follow convention.